styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteInputsUtils.java

```diff
@@ -0,0 +1,84 @@
+/*-
+ * -\-\-
+ * Spotify Styx Flyte Client
+ * --
+ * Copyright (C) 2016 - 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.flyte.client;
+
+import static flyteidl.core.Types.SimpleType.DATETIME_VALUE;
+
+import com.google.protobuf.Timestamp;
+import com.spotify.styx.util.ParameterUtil;
+import flyteidl.core.Interface;
+import flyteidl.core.Literals;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+
+public class FlyteInputsUtils {
+  private static final String PARAMETER_NAME = "parameter";
```

Comment:
can't this be styx_parameter?

Reply:
would it make sense to use the one we've been using so far? like `STYX_PARAMETER` ?